### DISCUSSION TITLE
lists and linebreaks

### DISCUSSION THREAD

**Discussion by derWalter** [Zettlr/Zettlr#4345] [General]
![image](https://github.com/Zettlr/Zettlr/assets/158642/d4766c6d-262b-4f4c-9e80-b157c98dcbd1)

This shows a testfile, where I was trying to see how lists get exported.
I expected that linebreaks get compiled as well, but failed to do so.

I wonder why a linebreak (when I hit return/enter, or shift+return/enter) doesn't translate to a '</ br>'

In Zettlr everything looks like i want it to, but as soon as I export it, all the linebreaks get translated into blanks/'&nbsp;'
and crushed into one line.
But this is also not consistent, as one can see on the screenshot. Sometimes it translates the break, sometimes it doesnt.
In some cases it also eats the headings (they have a blank at the end of the line, as stated in the documentation!)

what did I do wrong? Can this behavoir altered in any way? I guess using '</ br>' manually would hack around it,
but to me thats a very crude way to overcome the problem and nothing with which i would like to go on :P

**Comment by exanrauzer** [comment]
Markdown is a lightweight markup language commonly used for formatting text. In Markdown, a line break is indicated by inserting two spaces at the end of a line or by leaving a blank line between paragraphs. This is different from using HTML tags like </br> for line breaks.

When you're viewing your content in Zettlr, it likely applies its own rendering rules to display the line breaks as you expect them. However, when you export the content, it follows the standard Markdown rules, which may result in different behavior.

In Markdown, consecutive line breaks are interpreted as a single space, which is why you're seeing your text being condensed into one line when you export it. Markdown is designed to prioritize simplicity and readability in plain text, so it doesn't directly translate line breaks into HTML line break tags.

To overcome this behavior, you have a few options:

1. Use two spaces at the end of a line to create a line break in Markdown.
2. Insert a blank line between paragraphs to create a line break.
3. If you specifically want to use HTML line break tags, you can manually insert <br> where you want the line breaks to appear. However, this goes against the philosophy of Markdown and may not be the ideal solution.
4. To preserve line breaks without condensing the text into a single line, you can use the backslash character \ at the end of each line. This indicates that the line break should be preserved. This is known as a line continuation character.
5. Additionally, for non-numeric bullet point lists in Markdown, you can use either a hyphen -, a plus sign +, or an asterisk * followed by a space at the beginning of each list item.
